crates/isograph_schema/src/validate_schema.rs

```diff
@@ -1165,6 +1165,25 @@ pub fn get_missing_arguments<'a>(
         .collect()
 }
 
+pub fn get_provided_arguments<'a>(
+    argument_definitions: impl Iterator<Item = &'a ValidatedVariableDefinition> + 'a,
+    arguments: &[WithLocation<SelectionFieldArgument>],
+) -> Vec<ValidatedVariableDefinition> {
+    argument_definitions
+        .filter_map(|definition| {
+            let user_has_supplied_argument = arguments
```

Comment:
We didn't treat optional arguments and arguments with default values as missing. But should we? Seems like it's ok to allow users to pass optional arguments that are not provided as literals to `@loadable`